manifests/repo/yum.pp

```diff
@@ -33,15 +33,50 @@
     }
   }
 
+  case $facts['os']['name'] {
+    'RedHat': {
+      if $facts['os']['release']['major'] == '8' {
+        exec { 'Enabling Codeready-Builder repo':
+          command => 'dnf config-manager --set-enabled codeready-builder-for-rhel-8-x86_64-rpms',
+          path    => ['/usr/bin', '/usr/sbin',],
+          unless  => 'rpm -qa | grep -c python3-pyxattr',
```

Comment:
Not a yum/dnf user, but that does not look like the right way to check if a given repository is enabled.